docs/issue_template.md

```diff
@@ -0,0 +1,14 @@
+## User Story for the issue in the format of "As a user of type AAA, I need XXX so that I can YYY."
+
+
+## Details on issue or enhancement
+
+
+## For the development team
+-[] Issue user story documented
```

Comment:
You need a space before inside the braces so it shows up as a checklist
```md
- [ ] Issue user story documented
- [ ] UX input received
- [ ] Design completed
- [ ] Design validated by business team / UX
- [ ] Code review completed by peer
- [ ] Issue closing comment references any duplicate or connected issues or pull requests
- [ ] Issue closed
```
Then it renders as
- [ ] Issue user story documented
- [ ] UX input received
- [ ] Design completed
- [ ] Design validated by business team / UX
- [ ] Code review completed by peer
- [ ] Issue closing comment references any duplicate or connected issues or pull requests
- [ ] Issue closed